specification/datalake-store/resource-manager/Microsoft.DataLakeStore/stable/2016-11-01/account.json

```diff
@@ -630,12 +630,235 @@
+        },
+        "x-ms-examples": {
+          "Gets the specified Data Lake Store virtual network rule.": {
+            "$ref": "./examples/VirtualNetworkRules_Get.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "VirtualNetworkRules"
+        ],
+        "operationId": "VirtualNetworkRules_Update",
+        "description": "Updates the specified virtual network rule.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "name": "virtualNetworkRuleName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual network rule to update."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/UpdateVirtualNetworkRuleParameters"
+            },
+            "description": "Parameters supplied to update the virtual network rule."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the specified virtual network rule.",
+            "schema": {
+              "$ref": "#/definitions/VirtualNetworkRule"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Updates the specified virtual network rule.": {
+            "$ref": "./examples/VirtualNetworkRules_Update.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "VirtualNetworkRules"
+        ],
+        "operationId": "VirtualNetworkRules_Delete",
+        "description": "Deletes the specified virtual network rule from the specified Data Lake Store account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "name": "virtualNetworkRuleName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual network rule to delete."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully deleted the specified virtual network rule."
+          },
+          "204": {
+            "description": "The specified virtual network rule does not exist or was already deleted."
+          }
+        },
+        "x-ms-examples": {
```

Comment:
Is this a long running operation? if so model this operation as LRO.

Reply:
thanks. these vnet rule APIs are not async, so I think no need to model as LRO.